backend/src/wire/user.rs

```diff
@@ -47,6 +47,31 @@ pub(crate) struct UserContent {
     pub status: MembershipStatus,
```

Comment:
Some other optimisation: maybe smart to move the Membership status and roles of the user to the top level user and not the ˋContentˋ struct. I think, we could get rid of the ˋRegisterNewUserˋ struct then. But I might also miss some detail here.

Reply:
What about password in 'RegisterNewUser'?

Reply:
I forget this as well. Probably forget my suggestion.